go.mod

```diff
@@ -4,13 +4,12 @@ go 1.15
 
 require (
 	github.com/Azure/go-amqp v0.12.7
-	github.com/cloudevents/sdk-go/protocol/amqp/v2 v2.3.1-0.20210302080936-5c462007a5d5
-	github.com/cloudevents/sdk-go/v2 v2.3.1-0.20210302080936-5c462007a5d5
-	github.com/google/go-cmp v0.5.5
-	github.com/google/uuid v1.1.2
-	github.com/json-iterator/go v1.1.10
-	github.com/prometheus/client_golang v1.10.0
+	github.com/cloudevents/sdk-go/protocol/amqp/v2 v2.4.1
+	github.com/cloudevents/sdk-go/v2 v2.4.1
```

Comment:
do you know what are the changes in this version ?

Reply:
I don't know. This change is generated by `go mod tidy`.

Reply:
ok I will check and if they have updates, may be applied to the 4.9 branches.